src/main/resources/emissary/util/DateTimeFormats.cfg

```diff
@@ -0,0 +1,10 @@
+
+
+DATE_1 = "yyyy"
```

Comment:
Are we expecting a large list of DateTimeFormats? I was wondering if maybe more descriptive names would be helpful. 

Not really a suggestion, just a thought I had.

Reply:
I think we could have a large list. For some of these formats for the time being I agree it might make sense to have a more descriptive name. I'll add this.